pkgs/development/coq-modules/autosubst-ocaml/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  mkCoqDerivation,
+  coq,
+  version ? null,
+}:
+
+mkCoqDerivation {
```

Comment:
Why do you use `mkCoqDerivation`? This looks like a plain OCaml program: `buildDunePackage` might be more suitable.

Reply:
Hi @vbgl. Thanks for the review! Since it relates to specific coq version (specifically 8.19) and it depends on the coq package from opam in a normal setup, I thought it should use `mkCoqDerivation`. In fact, I struggled with `buildDunePackage` due to some coq dependency before I switched to `mkCoqDerivation`. But I am happy to change it to `buildDunePackage` and I just don't know what is the best practice. Would you like to help me change it?

Reply:
No, sorry. If it works as is, it is probably OK.